app/models/govuk_publishing_components/audit_applications.rb

```diff
@@ -55,6 +55,19 @@ def initialize(path, name)
             components: components_in_ruby,
           },
         ]
+
+        # applications might not all of these things for all components
```

Comment:
Small nitpick, should this read "# applications might not use all of these things for all components"